stock_release_channel_shipment_advice/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module is used to plan the  shipment advice for released pickings in a
+stock release channel.
+
+As this is the base module, it provides only a simple
+planning mode. All released pickings are grouped together in a single advice.
```

Comment:
Reading this, it is like all pickings inserted in the release channel are automatically planned in a shipment but I don't think so looking at the code.
Can you clarify how the shipment advice is supposed to be created in this simple mode and how the pickings are put in it?

Reply:
I updated the description, please check and let me know